lib/compiler_rt/README.md

```diff
@@ -261,6 +261,9 @@ Integer and Float Operations
 | ✓ | __getf2            | f128 | f128 | i32  | ..                             |
 | ✓ | __gexf2            | f80  | f80  | i32  | ..                             |
 | ✓ | __gekf2            | f128 | f128 | i32  | .. PPC                         |
+| ✓ | __aeabi_hcmpge     | f16  | f16  | i32  | .. ARM                         |
```

Comment:
Sorry, I also meant to place these before PPC, so things are somewhat sorted alphabetically. Compare this with the other usages of ARM symbols.